pbs_light.go

```diff
@@ -450,24 +452,32 @@ func sortBidsAddKeywordsMobile(bids pbs.PBSBidSlice, pbs_req *pbs.PBSRequest, pr
 			priceBucketStringMap := pbs.GetPriceBucketString(bid.Price)
 			roundedCpm := priceBucketStringMap[priceGranularitySetting]
 
+			width := strconv.FormatUint(bid.Width, 10)
+			height := strconv.FormatUint(bid.Height, 10)
+			hbSize := width + "x" + height
```

Comment:
what about returning width and height separately?

Reply:
I wanted to return `hb_size` as width x height to keep it consistent with how Prebid.js does it - http://prebid.org/adops.html